utils/zerovec/src/zerovec/mod.rs

```diff
@@ -433,10 +454,28 @@ where
                     let ptr = ptr as *mut P::ULE;
                     Vec::from_raw_parts(ptr, len, cap)
```

Comment:
Suggestion: There are now multiple places where we convert back and forth between a vec and its raw parts (here, `into_cow`, and `new_owned`). Consider making them share code or at least use the same style with similar safety comments.

Reply:
They already share code as much as possible; we deconstruct the vec in one place (new_owned) and construct it once (into_cow). There's also try_into_converted but that's about constructing a vec of a different type